sd-validation/test/shema-org-validation-test.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* eslint-env mocha */
+
+const assert = require('assert');
+const validateJSONLD = require('../');
+
+describe('schema.org validation', () => {
+  it('reports unknown types', async () => {
+    const errors = await validateJSONLD(`{
+      "@context": "http://schema.org",
+      "@type": "Cat"
+    }`);
+
+    assert.equal(errors.length, 1);
+    assert.equal(errors[0].message, 'Unrecognized schema.org type http://schema.org/Cat');
+  });
+
+  it('reports unknown types', async () => {
```

Comment:
either merge with previous `it` or slightly tweak the test name

Reply:
Fixed by Konrad.